.github/workflows/sync_docs.yaml

```
@@ -0,0 +1,19 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+name: Sync docs from Discourse
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: 50 00 * * *
```

```suggestion
    - cron: 20 02 * * *
```
Use the schedule listed in the [_sync_docs readme](https://github.com/canonical/data-platform-workflows/blob/main/.github/workflows/_sync_docs.md#nosql) to avoid overloading the Discourse API calls with all our repos.